stack/2020/SG-09-25-20.md

```diff
@@ -6,16 +6,17 @@
 - **When**: September 25th, 4pm-5pm UTC (September 25th, 9am-10am Pacific Daylight Time)
 - **Location**: *link on calendar invite*
 
-### Registration
 
-None required if you've attended before. Fill out the form here to sign up if
-it's your first time: https://forms.gle/Kvm62ak289uTQsft5. The meeting is open
-to CG members only.
+## Participants
 
-## Logistics
-
-The meeting will be on a zoom.us video conference.
-Installation is required, see the calendar invite.
+Francis McCabe
+Zalim Bashorov
+Alon Zakai
+Daniel Hillerström
+Emanuel Ziegler
+Jakob Kummerow
+Ryan Hunt
+Sam Lindley
```

Comment:
This list looks incomplete, TL, PS, HA, etc are not in the list of participants here.

Reply:
I was there too.

Reply:
Yep, I was there as well.

Reply:
Apologies for missing folk. Will try to add.